### ISSUE TITLE
Confusing error message on wake up timeout / long poll interval

### ISSUE THREAD

**Issue by jeroenr** [akka/reactive-kafka#426]
I'm getting the following warning: 

`KafkaConsumer poll is taking significantly longer (3000ms) to return from poll then the configured poll interval (5000ms). Waking up consumer to avoid thread starvation. `

When I read the message above it doesn't make much sense to me. My configured wake up time is 3000ms and my poll interval is 1000ms (poll timeout is 5000ms). Judging from the code it looks like the wake up timeout was exceeded, but I don't see how this is related to the poll interval.

My consumer code:

```
class MyConsumer(managerRef: ActorRef) extends Actor with ActorLogging {
  implicit val as = context.system
  implicit val ec = 
  ExecutionContext.fromExecutor(Executors.newFixedThreadPool(Config.kafka.`dedicated-threads`))
    implicit val am = {
      val decider: Supervision.Decider = {
        case t: Throwable =>
          log.error(t, "Consumer supervisor, stream failed.")
          self ! PoisonPill
          Stop
      }
      ActorMaterializer(ActorMaterializerSettings(as).withSupervisionStrategy(decider))
    }
  }

  implicit val timeout: Timeout = 5.seconds

  // consumer settings
  val consumerSettings = ConsumerSettings(as, new StringDeserializer, new ByteArrayDeserializer)
  val subscription = Subscriptions.topics("my_topic")

  Consumer.plainSource(consumerSettings, subscription)
    .map( record => {
       // deserialize record
     })
    .mapAsync(16)(model => (managerRef ? model).mapTo[Done])
    .runWith(Sink.ignore)
    .onComplete {
      case Success(_) =>
        // stop self to trigger restart
        log.error("Kafka consumer stream completed successfully, but it shouldn't have.")
        self ! PoisonPill
      case Failure(e) =>
        // stop self to trigger restart
        log.error(e, "Kafka consumer stream completed with failure.")
        self ! PoisonPill
    }

```

**Comment by patriknw** [comment]
I think you are right, would you be able to create a pull request fixing the log message?

**Comment by jeroenr** [comment]
Maybe it's worth as well to check somewhere that wakeupTimeout < pollTimeout?

**Comment by ennru** [comment]
Fixed with #437
